ksql-execution/src/main/java/io/confluent/ksql/execution/builder/KsqlQueryBuilder.java

```diff
@@ -135,6 +136,28 @@ public KsqlQueryBuilder withKsqlConfig(final KsqlConfig newConfig) {
         .push(context);
   }
 
+  @SuppressWarnings("unchecked")
+  public KeySerde<Object> buildKeySerde(
```

Comment:
As per comment in other PR the generics on this are wrong.

https://github.com/confluentinc/ksql/pull/3361#discussion_r325089990